C++/include/algorithm/dynamic_programming/README.md

```diff
@@ -44,3 +50,196 @@ Time               | Space
 _O(N<sup>3</sup>)_ | _O(N<sup>2</sup>)_
 
 where N is the number of matrices.
+
+## 2. Coin Change
+
+Given an unlimited amount of coins with different values, find the number of ways of making change for an given
+value using the coins.
+
+## Input
+- #### `coin`
+  Type: `vector<int> &`
+
+  It contains coins of different denomination.
+
+- #### `number_of_coins`
+  Type: `int`
+
+  It is the size of vector coin (coin.size()), or the total number of coins given.
+
+- #### `amount`
+  Type: `int`
+
+  It is the amount for which we have to find the number of ways to make change.
```

Comment:
As with above, "Represents the amount…" instead of "It is the amount…" Please change the rest of these, too.